### PR TITLE
Removed Android base code. Added the HMOS code for RevealLayout

### PR DESCRIPTION
**Pull request by Rejeesh2020**
Removed Android base code. Added the HMOS code for RevealLayout

### CODE REVIEW

**Review by sharathks84 — DISMISSED** [review]
OK, Approved

File: entry/build.gradle [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.huawei.ohos.hap'
+apply plugin: 'com.huawei.ohos.decctest'
+//For instructions on signature configuration, see https://developer.harmonyos.com/en/docs/documentation/doc-guides/ide_debug_device-0000001053822404#section1112183053510
+ohos {
+    compileSdkVersion 6
+    defaultConfig {
+        compatibleSdkVersion 4
```
Comment (sharathks84) [inline entry/build.gradle]: compatible sdk version should be 5

File: entry/src/main/resources/base/graphic/ic_launcher_background.xml [hunk 2]
```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:ohos="http://schemas.huawei.com/res/ohos"
```
Comment (sharathks84) [inline entry/src/main/resources/base/graphic/ic_launcher_background.xml]: animation is not working as per original lib

File: entry/src/main/resources/base/layout/ability_main.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<StackLayout xmlns:ohos="http://schemas.huawei.com/res/ohos"
+             xmlns:app="http://schemas.huawei.com/res-auto"
+             ohos:width="match_parent"
+             ohos:height="match_parent"
+             ohos:orientation="horizontal">
+
+    <per.goweii.reveallayout.RevealLayout
```
Comment (sharathks84) [inline entry/src/main/resources/base/layout/ability_main.xml:9]: Size of the components to be reduced and use english

File: entry/build.gradle (hunk 1 above)
Comment (Rejeesh2020) [reply entry/build.gradle]: Review comment reworked

File: entry/src/main/resources/base/graphic/ic_launcher_background.xml (hunk 2 above)
Comment (Rejeesh2020) [reply entry/src/main/resources/base/graphic/ic_launcher_background.xml]: Animation is not working as per original lib due to some API mismatch in HOS. Right now the same is kept as open and working in parallelly

File: entry/src/main/resources/base/layout/ability_main.xml (hunk 3 above)
Comment (Rejeesh2020) [reply entry/src/main/resources/base/layout/ability_main.xml:9]: Review comment reworked and kept same as orginal lib

**Review by sharathks84 — APPROVED** [review]
OK, approved

**Review by Gleelakrishna — APPROVED** [review]

**Pull request merged**